sway-core/src/ir_generation/function.rs

```diff
@@ -485,7 +485,9 @@ impl<'eng> FnCompiler<'eng> {
                 ..
             } => {
                 let enum_decl = self.engines.de().get_enum(enum_ref);
-                self.compile_enum_expr(context, md_mgr, &enum_decl, *tag, contents.as_deref())
+                let res =
```

Comment:
In other branches we do not use the temporary `res` variable. I don't see a benefit in having it here either and would go for the previous code without `res`.

Reply:
although in this particular instance I don't have a strong opinion either way, my preference is to not have review comments on code style, given that we have clippy to enforce standard practices. if there is a style preference that we want everyone to follow that clippy doesn't enforce, we should have a doc that lists them all. if a particular style requirement cannot be generalized to be defined in th doc, it shouldn't be requested in a review. that'll make reviews smoother without personal preferences coming in the way.

I re-emphasise, my point is, although related to the comment, not very specific to it. I meant to convey it for sometime now and found a place here.

Reply:
Seeing the review suggestions now in the thread - it really blows the comments unnecessary. Shame on me!

To the personal style, I also don't have strong preferences also in this case and don't think we need a separate doc. Slight style differences we have in code are not worth it.

What I do like to keep is the consistency of the code regardless of the preferences. In this particular case, why to introduce a purely stylistic change that is not related to the PR and breaks consistency with all the code around it? This I think makes sense to point out in the review.

But fully agree with you, definitely no need to bloat the review with 5 comments for that one tiny line! It could have been just one small remark.

Reply:
> it really blows the comments unnecessary. Shame on me!

 no no please, that's too strong. I only meant to point out about personal coding style preferences.

overall I think we're in agreement 🤝☺️

Reply:
In general I agree about code style comments, but in this particular instance the unnecessary local variable is a remnant from some debugging code I introduced, and I forgot to remove the local. I'll remove it.